src/apps/competitions/tests/test_submissions.py

```diff
@@ -213,3 +216,29 @@ def test_making_submission_to_phase_with_one_task_does_not_create_parents_or_chi
         assert resp.call_count == 1
         sub = Submission.objects.get(id=self.sub.id)
         assert not sub.has_children
+
+    def test_adding_task_to_phase_runs_submissions_on_new_task(self):
+
+        leaderboard = LeaderboardFactory()
+
+        for phase in self.comp.phases.all():
+            phase.leaderboard = leaderboard
+            phase.save()
```

Comment:
```suggestion
        self.comp.phases.all().update(leaderboard=leaderboard)
```
This is a nice, clean way to do this in one fell swoop.